osgp/shared/shared/src/main/java/org/opensmartgridplatform/shared/security/EncryptionTool.java

```diff
@@ -26,12 +26,15 @@ public EncryptionTool(final String secretKeyPath) throws FunctionalException {
     }
 
     public static void main(final String... args) throws Exception {
-        if (args.length != 2) {
+        if (args.length < 2) {
             System.out.println("Usage: java " + EncryptionTool.class.getCanonicalName() + " <path to secret key>"
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Replace this use of System.out or System.err by a logger. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://ci.opensmartgridplatform.org/sonarqube/coding_rules#rule_key=squid%3AS106)